src/Scaffolding/VS.Web.CG.Mvc/Templates/RazorPageGenerator/Bootstrap5/Edit.cshtml

```diff
@@ -47,7 +47,8 @@
             @:<div asp-validation-summary="ModelOnly" class="text-danger"></div>
         foreach (PropertyMetadata property in Model.ModelMetadata.Properties)
         {
-            var propertyRequiredText = property.IsRequired ? "<span class=\"text-danger\">*</span>" : string.Empty;
+            var propertyRequiredText = property.IsRequired ? "\r\n<span class=\"text-danger\">*</span>" : string.Empty;
+            var ariaRequiredAttributeHtml = property.IsRequired ? "required aria-required=\"true\"" : string.Empty;
```

Comment:
Can we get away with removing `required` and just doing `aria-required`? The `required` attribute interferes with custom form validation.

Reply:
(same suggestion for other similar changes in this PR)

Reply:
will try in a sec here

Reply:
yep it works without the `required`, removed